projects/examples/src/components/quick-search/quick-search-sync-async.example.module.ts

```diff
@@ -18,6 +18,11 @@ import {
     declarations: [QuickSearchSyncAsyncExampleComponent],
     exports: [QuickSearchSyncAsyncExampleComponent],
     entryComponents: [QuickSearchSyncAsyncExampleComponent],
-    providers: [ActionsSearchProvider],
+    providers: [
+        {
+            provide: ActionsSearchProvider,
+            useValue: new ActionsSearchProvider(false),
```

Comment:
Does this work? Doesn't the action search provider take in a TranslationService and a boolean?

Reply:
This is not the action search provider from the components package. It is created just for this example.